From: Christoph Hellwig <hch@lst.de>
Date: Sat, 12 May 2018 18:18:12 +0200
Subject: [PATCH] nvme.h: add support for the log specific field
Git-commit: 9b89bc3857a6c0dfda18ddae2a42c114ecc32753
Git-repo: git://git.kernel.dk/linux-block.git
Patch-Mainline: queued
References: bsc#1054245

NVMe 1.3 added a new log specific field to the get log page CQ
defintion, add it to our get_log_page SQ structure.

Signed-off-by: Christoph Hellwig <hch@lst.de>
Reviewed-by: Keith Busch <keith.busch@intel.com>
Reviewed-by: Sagi Grimberg <sagi@grimberg.me>
Reviewed-by: Martin K. Petersen <martin.petersen@oracle.com>
Reviewed-by: Hannes Reinecke <hare@suse.com>
Reviewed-by: Johannes Thumshirn <jthumshirn@suse.de>
---
 include/linux/nvme.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/nvme.h b/include/linux/nvme.h
index 2950ce957656..32f24f4ca763 100644
--- a/include/linux/nvme.h
+++ b/include/linux/nvme.h
@@ -880,7 +880,7 @@ struct nvme_get_log_page_command {
 	__u64			rsvd2[2];
 	union nvme_data_ptr	dptr;
 	__u8			lid;
-	__u8			rsvd10;
+	__u8			lsp; /* upper 4 bits reserved */
 	__le16			numdl;
 	__le16			numdu;
 	__u16			rsvd11;
-- 
2.12.3

